modules/manage/pages/recovery-mode.adoc

```diff
@@ -0,0 +1,201 @@
+
+In recovery mode, cluster configuration changes and other manual administrative actions are permitted so you can repair and restore the failed cluster. Topic reads and writes are disabled.
+
+== About recovery mode
+
+Recovery mode starts a cluster with limited functionality and deactivates normal traffic and data operations for a cluster. Redpanda does not load user-managed partitions on disk. This provides a more stable environment for troubleshooting issues and restoring the cluster to a usable state. 
+
+Recovery mode helps to prevent triggering a storm of partition leader elections and replication (such as increased learner replica recovery load after a long broker downtime) that may occur on startup. 
+
+Normal traffic and especially malicious traffic can disrupt availability for all users, including admin users. Recovery mode helps avoid placing the cluster under a heavy load of Kafka API and HTTP Proxy connections as you work to resuscitate the cluster.
+
+== Requirements
+
+To enable recovery mode, ensure that you have [rpk] installed, and you have local access to machines running Redpanda.
+
+Brokers can only enter recovery mode as it starts up, and not while it is already running. First set the broker configuration property to enable recovery mode, and then do a broker restart.
+
+== Use recovery mode
+
+To use recovery mode, first configure the broker to enable recovery mode when a broker starts.
+
+. Run the `rpk redpanda mode` command to set the `recovery_mode_enabled` broker configuration property to `true`.
+// TODO: confirm whether rpk reference is missing this update
++
+[,bash] 
+---- 
+rpk redpanda mode recovery
+----
+
+. Start the cluster. You can run `rpk cluster health` to check whether the cluster has entered recovery mode.
++
+It is possible to start a mixed-mode cluster, where some individual brokers are in recovery mode while others are not, but Redpanda recommends starting all brokers in the cluster in recovery mode together.
+
+All private Redpanda topics such as `__consumer_offsets` are accessible. While data in user-created topics is not available, you can manage the topics metadata.
+
+In recovery mode, you can also <<disable partitions,disable partitions>> if the cluster failure is localized to just a few partitions. If you disable specific partitions while in recovery mode, the disabled state persists even when you restart Redpanda again normally.
+
+You can perform the following operations in recovery mode:
+
+* Edit controller metadata
+** Edit cluster configuration properties
+** Add and remove users and ACLs
+** Add new brokers to the cluster
+** Modify topic properties
+** Delete topics
+** Delete WASM transforms
+* Edit consumer group metadata
+
+Node-wide and cluster-wide processes are disabled as they may disrupt recovery operations:
```

Comment:
IMO thls list belongs more to the "about recovery mode" section

Reply:
Agreed.  But also @ztlpn  isn't 'edit controller metadata' the sum of the items below (and we could skip that bullet)?

Reply:
just checking implementation.. @rockwotj is the wasm transforms internal topic part of the internal topics list such that this will work (deleting wasm transforms)?

Reply:
> But also @ztlpn isn't 'edit controller metadata' the sum of the items below (and we could skip that bullet)?

What is listed above under "edit controller metadata" are all user actions. What is listed below are internal Redpanda processes.